rest-server/src/config/db.ts

```diff
@@ -0,0 +1,34 @@
+import * as mysql from 'mysql';
+import * as Promise from 'bluebird';
+
+import {
+  success,
+  error
+} from '../lib/log'
+
+interface Option {
+  db: string;
+}
+
+const options: Option = {
+  db: process.env.NODE_ENV === 'test' ? process.env.DATABASETEST : process.env.DATABASE
+}
+
+const dataBase: any = mysql.createConnection({
+  host: 'localhost',
```

Comment:
all this should get read from process.env


process.env.Localhost
process.env.password
process.env.user